icubam/db/synchronizer.py

```diff
@@ -147,13 +175,23 @@ def sync_bed_counts(self, bedcounts_df, user=None):
 
 class CSVSynchcronizer(StoreSynchronizer):
   """Ingests CSV TextIO objects into datastore."""
+  def __init__(self, store_db, finess):
+    super().__init__(store_db)
+    self.finess = finess
+
   def sync_icus_from_csv(self, csv_contents: TextIO, force_update=False):
     """Check that columns correspond, insert into a DF and sychronize."""
-    icus_df = pd.read_csv(csv_contents)
+    try:
+      finess_contents = open(self.finess, "r", encoding="utf-8")
+      finess_df = pd.json_normalize(json.load(finess_contents))
```

Comment:
Maybe add an assertion here, that finess_df.shape[1] > 1 or something similar?

Reply:
Done.